futures-util/Cargo.toml

```diff
@@ -34,8 +34,8 @@ futures-sink-preview = { path = "../futures-sink", version = "=0.3.0-alpha.16",
 futures-select-macro-preview = { path = "../futures-select-macro", version = "=0.3.0-alpha.16", default-features = false, optional = true }
 proc-macro-hack = { version = "0.5", optional = true }
 proc-macro-nested = { version = "0.1.2", optional = true }
-rand = { version = "0.6.4", optional = true }
-rand_core = { version = ">=0.2.2, <0.4", optional = true } # See https://github.com/rust-random/rand/issues/645
+rand = { version = "0.6.5", optional = true }
+rand_core = { version = ">=0.4, <0.5", optional = true } # See https://github.com/rust-random/rand/issues/645
```

Comment:
This should be just `0.4` now.

Reply:
Yes, `0.4.1` has been yanked.

Reply:
I just meant that `0.4` is an equivalent bound to `>=0.4, <0.5`.

Reply:
Oh, I was thinking that somehow I wrote `">=0.4.1, <0.5"` and opened this PR.